src/actinia_core/rest/persistent_processing.py

```diff
@@ -502,16 +633,15 @@ def _merge_mapset_into_target(self, source_mapset, target_mapset):
 
             if os.path.exists(source_path) is True:
                 if directory == "group":
-                    group_dirs = os.listdir(source_path)
-                    for group_dir in group_dirs:
-                        group_file = os.path.join(source_path, group_dir, "REF")
-                        if os.path.isfile(group_file):
-                            for line in fileinput.input(group_file, inplace=True):
-                                print(line.replace(
-                                    source_mapset, target_mapset), end='')
-                        else:
-                            raise AsyncProcessError("group %s has no REF file"
-                                                    % (group_dir))
+                    self._change_mapsetname_in_group(
+                        source_path, source_mapset, target_mapset)
+                if directory == "tgis":
+                    target_tgis_db = None
+                    if os.path.isdir(os.path.join(target_path, 'tgis')):
```

Comment:
It might be better to check for the actual existence of `target_path/tgis/sqlite.db` - I don't know if there can be a case that the folder `tgis` exists, but not the `tgis/sqlite.db`

Reply:
I don't think so, only if the file is manually deleted. I tested it locally - I had one STRDS and removed it. Folder `tgis` and file `sqlite.db`  both still exist, but table `strds_base` is empty.